catalog/ui/search-ui/standard/src/main/webapp/js/controllers/cesium.controller.js

```diff
@@ -86,6 +102,18 @@ define(['application',
                     }
                 );
 
+                var that = this;
+                viewer.camera.moveEnd.addEventListener(function() {
+                    if(that.clustering) {
+                        var cartographic = new Cesium.Cartographic();
+                        var metersToKm = 0.001;
+                        viewer.scene.mapProjection.ellipsoid.cartesianToCartographic(viewer.camera.positionWC, cartographic);
+                        var cameraHeight = (cartographic.height * metersToKm).toFixed(1);
```

Comment:
Do we need to make sure `cameraHeight` does not go under a threshold?  I know for bounding box drawing we make sure the distance between points is greater than EPSILON6 from Cesium.

https://github.com/codice/ddf/blob/9b4ea7781035426eb1571344846eff3cfd1f52fe/catalog/ui/search-ui/standard/src/main/webapp/js/widgets/cesium.bbox.js#L63